src/components/Verse/VerseText.tsx

```diff
@@ -54,7 +54,7 @@ const VerseText = ({ words, isReadingMode = false, isHighlighted }: VerseTextPro
   const isTajweedFont = quranFont === QuranFont.Tajweed;
   const isBigTextLayout =
     (isReadingMode &&
-      (quranTextFontScale > 3 || showWordByWordTranslation || showWordByWordTransliteration)) ||
```

Comment:
3 was not chosen because it's the middle but rather it's the maximum font-scale we can apply using flex and above that the words will over-flow out of the QuranReader on mobile